dateparser/custom_language_detection/langdetect.py

```diff
@@ -0,0 +1,38 @@
+from langdetect.detector_factory import DetectorFactory, PROFILES_DIRECTORY
+import langdetect
```

Comment:
```suggestion
from langdetect.detector_factory import DetectorFactory, PROFILES_DIRECTORY

import langdetect
```

After sorting/optimizing imports.

Reply:
Hi @kishan3 should there be space as they are imported from same library

Done 👍🏻